### ISSUE TITLE
Generated CMake file hardcodes path to librt.so

### ISSUE THREAD

**Issue by pitrou** [google/benchmark#824]
The file `build/src/CMakeFiles/Export/lib/cmake/benchmark/benchmarkTargets.cmake` which is generated when building gbenchmark contains the following code on Linux:
```cmake
set_target_properties(benchmark::benchmark PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include"
  INTERFACE_LINK_LIBRARIES "-pthread;/usr/lib/x86_64-linux-gnu/librt.so"
)
```

Instead of hard-coding the path to `librt.so` it should probably use `-lrt`. This is causing issues for conda packages (see https://github.com/conda-forge/benchmark-feedstock/pull/5#issuecomment-496183792).

**Comment by dominichamon** [comment]
as far as i can tell this is coming from
https://github.com/google/benchmark/blob/33d440465041fe83b831f537b9646756655806f0/src/CMakeLists.txt#L32

ie, we use find_library from cmake to find rt.

Dominic Hamon | Google
*There are no bad ideas; only good ideas that go horribly wrong.*

**Comment by mingwandroid** [comment]
This is something I can probably be useful helping to look into along with the Cmake developers. @pitrou, if you can open something at anaconda-issues I can assign it to myself. Thanks.

**Comment by sergiud** [comment]
@dominichamon This was fixed in #1244.
